diff --git a/node_modules/bootstrap-vue/esm/utils/props.js b/node_modules/bootstrap-vue/esm/utils/props.js
index 5da0ede..24ef7ee 100644
--- a/node_modules/bootstrap-vue/esm/utils/props.js
+++ b/node_modules/bootstrap-vue/esm/utils/props.js
@@ -105,5 +105,5 @@ var configurablePropDefaultFnName = makePropConfigurable({}, '', '').default.nam
 // and isn't the props default function
 
 export var hasPropFunction = function hasPropFunction(fn) {
-  return isFunction(fn) && fn.name !== configurablePropDefaultFnName;
+  return isFunction(fn) && fn.name && fn.name !== configurablePropDefaultFnName;
 };
diff --git a/node_modules/bootstrap-vue/src/utils/props.js b/node_modules/bootstrap-vue/src/utils/props.js
index 62eaa6b..f40bbe4 100644
--- a/node_modules/bootstrap-vue/src/utils/props.js
+++ b/node_modules/bootstrap-vue/src/utils/props.js
@@ -94,4 +94,4 @@ const configurablePropDefaultFnName = makePropConfigurable({}, '', '').default.n
 
 // Detect wether the given value is currently a function
 // and isn't the props default function
-export const hasPropFunction = fn => isFunction(fn) && fn.name !== configurablePropDefaultFnName
+export const hasPropFunction = fn => isFunction(fn) && fn.name && fn.name !== configurablePropDefaultFnName
